examples/basic/main.tf

```diff
@@ -140,24 +74,57 @@ module "vpc" {
   version            = "1.60.0"
   name               = "test-vpc"
   cidr               = "10.0.0.0/16"
-  azs                = ["${data.aws_availability_zones.available.names[0]}", "${data.aws_availability_zones.available.names[1]}", "${data.aws_availability_zones.available.names[2]}"]
```

Comment:
I wonder if this always failed when run against regions where only 2 AZs are available.